scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -567,6 +594,7 @@ class FossId internal constructor(
      * Delete a scan with [scanCode].
      */
     private suspend fun deleteScan(scanCode: String) {
+        log.info { "Deleting scan $scanCode." }
```

Comment:
In the other place where the function is called we already do `log.warn("Deleting previous scan $it.")`.
Therefore you should also delete latter logging code.

Reply:
Also pass the `Level` and use `log.log` to still be able to do it at `WARN` level in the other case and `INFO` in this one.